release-content/0.16/release-notes/virtual_geometry_improvements.md

```diff
@@ -0,0 +1,7 @@
+Virtual geometry (the `meshlet` feature) got some performance improvements in Bevy 0.16.
+
+Read more details on the [author's blog post](https://jms55.github.io/posts/2025-03-27-virtual-geometry-bevy-0-16).
+
+Users are not required to regenerate their `MeshletMesh` assets, but doing so is recommended in order to take advantage of the improved clustering algorithm.
```

Comment:
Link for MeshletMesh please? Feel free to use a dev-docs link; I can quickly find-and-replace at the end.